Defend The Divine/Assets/Scripts/Towers/TowerPlacement.cs

```diff
@@ -22,28 +22,19 @@ void Update()
     {
         GameManager manager = GameManager.Instance;
```

Comment:
This isn't very good, creating & setting manager every frame is inefficient. There's not any difference between using GameManger.Instance or saving it to manager, just removes a little typing. If you want to use manager like this, set it in Start()